db/seeds.rb

```diff
@@ -48,12 +36,87 @@
   [ "H-2.3", floor, "Bachelorprojekt Baudisch"],
   [ "A-1.15", floor2, "A-1.15"]
 ]
+hs_rooms = [
+  [floor, "HS 3", 1],
+  [floor, "Passage", 2],
+  [floor, "HS 2", 3],
+  [floor, "Foyer", 4],
+  [floor, "R1", 5],
+  [floor, "R2", 6],
+  [floor, "R3", 7],
```

Comment:
```suggestion
  [floor, "Passage", 2],
  [floor, "HS 2", 3],
  [floor2, "Foyer", 4],
  [floor2, "R1", 5],
  [floor, "R2", 6],
  [floor, "R3", 7],
```
Please use both first and second floor (we need to test smth)